cybered/shared/templates/includes/footer.html

```diff
@@ -1,10 +1,40 @@
-<div class="mb-6">
+<div style="padding-bottom: 9rem !important;" id="bottom_navbar_padding">
```

Comment:
So, I know that having style and scripts inline is kind of bad form; but it seems like it makes sense in this case because the footer is kind of a self-contained thing, and it doesn't make a lot of sense IMO to have this as external css and javascript.
Thoughts?